homeassistant/components/nina/__init__.py

```diff
@@ -0,0 +1,98 @@
+"""The Nina integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+from typing import Any
+
+from async_timeout import timeout
+from pynina import ApiError, Nina, Warning as NinaWarning
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import (
+    _LOGGER,
+    ATTR_EXPIRES,
+    ATTR_HEADLINE,
+    ATTR_ID,
+    ATTR_SENT,
+    ATTR_START,
+    CONF_REGIONS,
+    CORONA_FILTER,
+    DOMAIN,
+    SCAN_INTERVAL,
+)
+
+PLATFORMS: list[str] = ["binary_sensor"]
```

Comment:
Since today there is now a Platform StrEnum that you can use instead of this. `Platform.BINARY_SENSOR`